contrib/flare/template.py

```diff
@@ -0,0 +1,51 @@
+pkgname = "flare"
+pkgver = "0.15.0"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "blueprint-compiler",
+    "cargo-auditable",
+    "pkgconf",
+    "protoc",
+    "meson",
+    "gettext",
+    "desktop-file-utils",
+]
+makedepends = [
+    "cairo-devel",
+    "gdk-pixbuf-devel",
+    "glib-devel",
+    "gtk4-devel",
+    "gtksourceview-devel",
+    "libadwaita-devel",
+    "pango-devel",
+    "libspelling-devel",
+]
+pkgdesc = "Unofficial Signal Desktop app"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "AGPL-3.0-only"
+url = "https://mobile.schmidhuberj.de/flare"
+source = f"https://gitlab.com/schmiddi-on-mobile/flare/-/archive/{pkgver}/flare-{pkgver}.tar.gz"
+sha256 = "6bd9fe77f9e40fc3d081baf5bd5665b94b3514c5cc488305e46d2656b1f68f7b"
+
+
+def prepare(self):
+    from cbuild.util import cargo
+
+    self.make_dir = ""
+    cargo.Cargo(self).vendor()
+    self.make_dir = "build"
+
+
+def init_build(self):
+    from cbuild.util import cargo
+
+    self.make_dir = ""
+    renv = cargo.get_environment(self)
+    self.make_env.update(renv)
+    self.make_dir = "build"
```

Comment:
This is horrible, but if I don't get rid of the `make_dir` for these steps, it fails to build. If I remove the `make_dir` for the whole template, it also fails to build. Any ideas?